.travis.yml

```diff
@@ -9,18 +9,22 @@ dart:
 - dev
 # See https://docs.travis-ci.com/user/languages/dart/ for details.
 dart_task:
-- test: --platform vm
-# Uncomment this line to run tests on the browser.
-# - test: --platform firefox
+  - dartfmt
+  - dartanalyzer
+  # TODO: reinstate tests once https://github.com/dart-lang/http_retry/issues/6 is fixed
+  #  - test: --platform vm
+  #    xvfb: false
+  #  # Uncomment this line to run tests on the browser.
+  #  - test: --platform firefox
 
 # Only run one instance of the formatter and the analyzer, rather than running
 # them against each Dart version.
 matrix:
-  include:
-  - dart: dev
-    dart_task: dartfmt
-  - dart: dev
-    dart_task: dartanalyzer
+  exclude:
```

Comment:
There is no stable, just remove the matrix

Reply:
Done